### PR TITLE
EVG-1924 correctly link multiple executions in host event log

### PR DESCRIPTION
**Pull request by john-m-liu**

### CODE REVIEW

File: model/event/host_event_test.go [hunk 1]
```diff
@@ -147,6 +147,21 @@ func TestLoggingHostEvents(t *testing.T) {
 			So(eventData.Hostname, ShouldBeBlank)
 			So(eventData.TaskId, ShouldBeBlank)
 			So(eventData.TaskPid, ShouldEqual, taskPid)
+
+			// test logging multiple executions of the same task
+			err = UpdateExecutions(hostId, taskId, 0)
+			So(err, ShouldBeNil)
+
+			eventsForHost, err = Find(AllLogCollection, HostEventsInOrder(hostId))
+			So(err, ShouldBeNil)
```
Comment (tychoish) [inline model/event/host_event_test.go]: add an assertion that there should be at least one result?

File: model/event/host_event.go [hunk 2]
```diff
@@ -112,3 +116,22 @@ func LogHostTeardown(hostId, teardownLogs string, success bool, duration time.Du
 func LogMonitorOperation(hostId string, op string) {
 	LogHostEvent(hostId, EventHostMonitorFlag, HostEventData{MonitorOp: op})
 }
+
+// UpdateExecutions updates host events to track multiple executions of the same task
+func UpdateExecutions(hostId, taskId string, execution int) error {
+	const ResourceIdKey = "r_id"
+	const TaskIdKey = "t_id"
+	const DataKey = "data"
+	const ExecutionKey = "execution"
```
Comment (tychoish) [inline model/event/host_event.go]: it's weird to have public values in function scopes. 

also these keys should be declared/checked with bsonutil.MustHaveTag() in the package scope (I hate this, but, alas, it's the pattern. 

also you can cosnt ( <names> = <values>

Comment (john-m-liu) [reply model/event/host_event.go]: Oops, I was confused about what package I was in when I did this. It should be cleaned up now

File: model/event/host_event_test.go (hunk 1 above)
Comment (john-m-liu) [reply model/event/host_event_test.go]: Done

**Pull request merged**